bin/tilt

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
It feels like you have plenty of files in bin, that should not be part of this project.